src/config.rs

```diff
@@ -186,6 +191,7 @@ mod instance {
         }
     }
 
+    #[allow(clippy::unnecessary_wraps)]
```

Comment:
This should be addressed

Reply:
Clippy is "wrong" here; while yes, right now this can't return an error I do very much want to implement some checks here later, e.g. if the configuration contains no mechanisms.